tests/ut_callmodel/callmodeltest.cpp

```diff
@@ -1024,12 +1003,13 @@ void CallModelTest::testModifyEvent()
     QCOMPARE(e2.direction(), Event::Inbound);
 }
 
-// Make sure that phone numbers resolve to the right contacts even if they
-// minimize to the same number.
+// Test that phone numbers resolve to the same contact if they minimize
```

Comment:
I think we should get a second opinion on this from @pvuorela and @Tomin1  -- our minimisation algorithm isn't exactly perfect, so the currently enforced behaviour is likely to result in some collisions.  I tend to think that those are ... expected, and unless we want to improve minimisation (or minimised-comparison) using libphonenumber I think we should just bite the bullet and do it this way.  But, having some extra discussion around this point would be good, I think.